src/components/atoms/HeaderCell/HeaderCell.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import HeaderCell from './HeaderCell';
+
+
+test('renders text ', () => {
+    const tableRow = document.createElement('tr');
+    render(<HeaderCell>Name</HeaderCell>, {
+        container: document.body.appendChild(tableRow)
+    });
+    const heading = screen.getByText('Name');
+    expect(heading).toBeInTheDocument();
+});
```

Comment:
Something I usually do is wrap my test suit in a describe block with the component name so that in the console it's clear which tests belong to which component.